app/assets/javascripts/controllers/container_dashboard/util/charts-mixin.js

```diff
@@ -49,6 +83,36 @@ angular.module('miq.util').factory('chartsMixin', function() {
       units    : __('KBps'),
       dataName : __('KBps'),
       tooltipFn  : dailyTimeTooltip
+    },
+    dailyPodUsageConfig: {
+      chartId     : 'podUsageDailyChart',
+      headTitle   : __('Pod Utilization Trends'),
```

Comment:
@yaacov also this title is not very clear. I think it should be just "Pod Trends". In fact there's no detail about utilization in the graph.